contrib/ohi-release/action.yml

```diff
@@ -0,0 +1,74 @@
+name: OHI release
+description: Wrapper for release toolkit that runs commands needed to release an OHI
+inputs:
+  excluded-dirs:
+    description: 'Exclude commits whose changes only impact files in specified dirs relative to repository root. Defaults to ".github".'
+    required: true
+    default: '.github'
+  fail-if-held:
+    description: 'Fail if the held toggle is active'
+    required: true
+    default: 'true'
+  link-dependencies-enable:
+    description: 'Enables/disables link dependencies step'
+    required: true
+    default: 'false'
+  link-dependencies-dictionary:
+    description: 'Sets the link dependency dictionary. Defaults to ".github/rt-dictionary.yaml".'
+    required: true
+    default: '.github/rt-dictionary.yaml'
+outputs:
+  next-version:
+    description: "Version of this release"
+    value: ${{ steps.version.outputs.next-version }}
+  release-title:
+    description: "Title of this release"
+    value: ${{ steps.release.outputs.title }}
+  release-changelog:
+    description: "Complete changelog of this release"
+    value: ${{ steps.release.outputs.changelog }}
+  release-changelog-partial:
+    description: "Changelog for only this release"
+    value: ${{ steps.release.outputs.changelog-partial }}
+runs:
+  using: composite
+  steps:
+    - uses: newrelic/release-toolkit/validate-markdown@v1
```

Comment:
This is not required here, the validate-markdown is supposed to be executed as a separate action on the PR workflow or individually executing rt validate-markdown in your local environment.
We might add it here as an extra validation if you decide is valuable, because it might be useful to have it if executing the shell locally.

Reply:
I found issues creating the markdown way below because the markdown was not valid and this allows the user to not footgun itself.

Here is a test case:
Given a Changelog (snippet) like this
```markdown
## Unreleased
### Changes
test
```
If I try to `validate-markdown` it returns `unreleased changelog can't only contain notes`

But if I run `generate-yaml` it returns a valid YAML:
```yaml
notes: |-
    ### Changes
    test
changes: []
dependencies: []
```

There are lots of tests that I am able to pass, like using this markdown:
```markdown
## Unreleased
### Changes
test
### Enhancement
- A test in singular
### Enhancements
- A test in plural
```

But is a safety measurement until we add better testing when we footgun ourselves a couple of times 🤣